doc/source/howtos/crash-recovery.rst

```diff
@@ -0,0 +1,295 @@
+re-connect to it. After several timeouts, node A is removed from the
+cluster. The quorum is saved (2 out of 3 nodes are up), so no service disruption
+happens. After it is restarted, node A joins automatically (as described in
+:ref:`pxc-crash-recovery-scenario-1`).
+
+.. _pxc-crash-recovery-scenario-5:
+
+Scenario: Two nodes disappear from the cluster
+================================================================================
+
+Two nodes are not available and the remaining node (node C) is not able to form
+the quorum alone. The cluster has to switch to a non-primary mode, where MySQL
+refuses to serve any SQL queries. In this state, the :command:`mysqld` process
+on node C is still running and can be connected to but any statement related to
+data fails with an error
+
+.. code-block:: guess
+
+   mysql> select * from test.t1;
+   ERROR 1047 (08S01): Unknown command
+
+Reads are possible until node C decides that it cannot access node A and
+node B. New writes are forbidden immediately thanks to the certification based
+replication in Galera. The log of node C reports the following
+
+.. code-block:: text
+
+   140814 0:42:13 [Note] WSREP: commit failed for reason: 3
+   140814 0:42:13 [Note] WSREP: conflict state: 0
+   140814 0:42:13 [Note] WSREP: cluster conflict due to certification failure for threads:
+   140814 0:42:13 [Note] WSREP: Victim thread:
+   THD: 7, mode: local, state: executing, conflict: cert failure, seqno: -1
+   SQL: insert into t values (1)
+
+As soon as the other nodes become available, the cluster is formed again
+automatically. If node B and node C were just network-severed from
+node A, but they can still reach each other, they will keep functioning as
+they still form the quorum.
+
+If node A and node B crashed, you need to do enable the primary component on
+node C manually, before you can bring up node A and node B. The command to do
+this is:
+
+.. code-block:: guess
+
+   SET GLOBAL wsrep_provider_options='pc.bootstrap=true';
+
+This approach only works if the other nodes are down before doing that!
+Otherwise, you end up with two clusters having different data.
+
+
+.. admonition:: Cross Reference
+
+   :ref:`add-node`
+
+.. _pxc-crash-recovery-scenario-6:
+
+Scenario: All nodes went down without a proper shutdown procedure
+================================================================================
+
+This scenario is possible in case of datacenter power failure or when hitting a
+MySQL or Galera bug. Also it may happen as a result of data consistency being
+compromised where the cluster detects that each node has different data. The
+:file:grastate.dat` file is not updated and does not contain a valid sequence
+number (seqno). It may look like this:
+
+.. code-block: text
+
+   cat /var/lib/mysql/grastate.dat
+   # GALERA saved state
+   version: 2.1
+   uuid: 220dcdcb-1629-11e4-add3-aec059ad3734
+   seqno: -1
+   cert_index:
```

Comment:
isn't there also a safe_to_bootstrap variable?

Reply:
Added the description of this feature based on Galera Cluster documentation